src/contact.ts

```diff
@@ -55,9 +64,8 @@ export type ContactQueryFilter = {
 
 /**
  * Class Contact
- * blabla...
- * **IMPORTANT**
  *
+ * `Contact` is `Sayable`
```

Comment:
I don't think we should add each method, some method like `Contact.name()`, the code already can explain itself as `Gender` you said.

I have added method `Contact.find()` in this commit.

But your suggestion make sense, except method `find`, I think I should add the following method:
- avatar()
- say()

What other functions do you think I should add too?